core/tests/ui/e2e/order.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { expect, test } from '~/tests/fixtures';
+
+function formatAmount(amount: number): string {
+  return new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+  }).format(amount);
+}
+
+function formatDate(s: string): string {
+  return new Date(s).toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+}
+
+test('Orders page has empty state', async ({ page, account }) => {
+  const customer = await account.create();
+
+  await customer.login();
+
+  await page.goto('/account/orders/');
+
+  await expect(page.getByRole('heading', { name: 'Orders' })).toBeVisible();
+  await expect(page.getByText('No orders yet')).toBeVisible();
+});
+
+test('Order details are visible on Orders page', async ({ page, account, order }) => {
+  const customer = await account.create();
+
+  await customer.login();
+
+  const orderDetails = await order.create(88, customer.id);
```

Comment:
🍺 If we can create an order through the UI like the end user instead of a call, we should do that but since I don't know if that feature is fully functional this is fine for now. Otherwise, it feels not completely e2e testing to me.

Reply:
I totally see your point. Since we already have a test, where user adds product to the cart, navigates to checkout and places order - I considered this path as covered. 
I was thinking about creating an order via UI here, but it seems to be too long user journey for repeating it in every order related test.